WalletWasabi/Tor/TorProcessManager.cs

```diff
@@ -131,30 +139,64 @@ public async Task<bool> StartAsync(CancellationToken token = default)
 			return false;
 		}
 
-		public async Task StopAsync()
+		/// <summary>
+		/// Connects to Tor control using a TCP client or throws <see cref="TorControlException"/>.
+		/// </summary>
+		/// <exception cref="TorControlException">When authentication fails for some reason.</exception>
+		/// <seealso href="https://gitweb.torproject.org/torspec.git/tree/control-spec.txt">This method follows instructions in 3.23. TAKEOWNERSHIP.</seealso>
+		private async Task<TorControlClient> InitTorControlOrThrowAsync(CancellationToken token = default)
```

Comment:
nit: Do we need that OrThrow in the method name? I think it is obvious.

Reply:
You are right. In earlier iterations of this PR the name made more sense but now it does not really. Will change.